src/script/components/MessagesList/Message/ContentMessage/asset/ImageAsset/ImageAsset.tsx

```diff
@@ -120,7 +119,7 @@ export const ImageAsset = ({
   };
 
   const imageStyle: CSSObject = {
-    width: imageWidth,
+    width: 'max-content',
```

Comment:
**question:** Wouldn't it always make the image full parent's width? Even if we want to send a small image? E.g 50x50px will go full screen size?

Reply:
Very valid question +1

Reply:
No, because it is not used as the width: on the element, instead it was used for max-width:
So smaller images would still display smaller.

But thinking about it, it would overflow. fit-content is what should really be used here. Since it adjusts the image to fit the outer div instead of overflowing